app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -0,0 +1,58 @@
+@import "design-patterns/buttons";
+@import "mixins/media-down";
+@import "mixins/margins";
+
+%gem-c-button,
+.gem-c-button {
+  @include button;
+
+  @media(max-width: 425px) {
+    box-sizing: border-box;
+    width: 100%;
+    text-align: center;
+  }
+
+  &:focus {
+    outline: 3px solid $focus-colour;
+  }
+}
+
+// this will be moved and extended into a model for general component spacing
```

Comment:
Is this comment still correct?

Reply:
It is. Funnily enough we were just talking about this earlier today. The current situation is that there's no urgency to implement this model and we'll probably look at it properly when the design system stuff starts to be integrated. In the meantime there's probably some work to be done to ensure that those components that do implement options for spacing do so in a consistent way (they might already, but we've yet to look).

Reply:
Could change the comment to reflect your comments here?